src/components/color-picker-swatch/color-picker-swatch.tsx

```diff
@@ -1,9 +1,13 @@
-import { Component, Element, h, Prop, VNode, Watch } from "@stencil/core";
+import { Component, Element, Fragment, h, Prop, VNode, Watch } from "@stencil/core";
 import Color from "color";
 import { getModeName } from "../../utils/dom";
 import { Scale } from "../interfaces";
+import { hexify } from "../color-picker/utils";
 import { COLORS, CSS } from "./resources";
 
+const CHECKER_SQUARE_SIZE_IN_PX = 4;
```

Comment:
nitpick: Should these const be moved to the resources file? Incase need exporting for tests/etc.